Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -307,7 +307,7 @@ public function clearEmptyTokens()
         $limit = $this->count();
         $index = 0;
```

Comment:
Then it would suggest that `$index` is used for that one loop, but in fact it is reused after and it rely on the state after all iterations.